providers/implementations/keymgmt/dh_kmgmt.c

```diff
@@ -306,6 +317,7 @@ static const OSSL_PARAM dh_params[] = {
     OSSL_PARAM_int(OSSL_PKEY_PARAM_BITS, NULL),
     OSSL_PARAM_int(OSSL_PKEY_PARAM_SECURITY_BITS, NULL),
     OSSL_PARAM_int(OSSL_PKEY_PARAM_MAX_SIZE, NULL),
+    OSSL_PARAM_octet_string(OSSL_PKEY_PARAM_TLS_ENCODED_PT, NULL, 0),
```

Comment:
Is this param exportable?

Reply:
No. It just a TLS specific way of encoding a public key. Once set we just treat it like a normal public key and export it the same as any other.